packages/components/pie-chip/test/component/pie-chip.spec.ts

```diff
@@ -79,4 +83,171 @@ test.describe('PieChip - Component tests', () => {
             await expect(closeButton).toBeDisabled();
         });
     });
+
+    test.describe('Chip Close: aria: attribute', () => {
+        test.describe('aria-label', () => {
+            test.describe('when an aria close value is provided', () => {
+                test('should render on the component with the correct value', async ({ mount, page }) => {
+                    // Arrange
+                    await mount(PieChip, {
+                        props: {
+                            ...dismissibleProps,
+                            aria: {
+                                close: 'Chip Close',
+                            },
+                        } as ChipProps,
+                    });
+
+                    const closeButton = page.locator(closeBtnSelector);
+
+                    // Act && Assert
+                    await expect(closeButton).toHaveAttribute('aria-label', 'Chip Close');
+                });
+            });
+
+            test.describe('when an aria close value is not provided', () => {
```

Comment:
I would say that there is still a value being provided here, it's just an empty string

Reply:
Yes good catch!